middle/metrology.owl

```diff
@@ -171,12 +171,28 @@ ISO 80000-1</rdfs:comment>
     <!-- http://emmo.info/emmo/middle/metrology#EMMO_08b308d4_31cd_4779_a784_aa92fc730f39 -->
 
     <owl:Class rdf:about="http://emmo.info/emmo/middle/metrology#EMMO_08b308d4_31cd_4779_a784_aa92fc730f39">
-        <annotations:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9>Derived units are defined as products of powers of the base units corresponding to the relations defining the derived quantities in terms of the base quantities.</annotations:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9>
+        <annotations:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 xml:lang="en">Derived units are defined as products of powers of the base units corresponding to the relations defining the derived quantities in terms of the base quantities.</annotations:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9>
         <skos:prefLabel xml:lang="en">DerivedUnit</skos:prefLabel>
     </owl:Class>
     
 
 
+    <!-- http://emmo.info/emmo/middle/metrology#EMMO_0a318776_b067_4de0_a2a6_cba2cf6333f8 -->
```

Comment:
IdiomaticSymbol should be in perceptual.owl, not metrology.owl. Needs an elucidation.